dev/ws/go.mod

```diff
@@ -0,0 +1,53 @@
+module github.com/styrainc/regal/dev/ws
+
+go 1.24.4
+
+replace github.com/styrainc/regal => ../..
+
+require (
+	github.com/gorilla/websocket v1.5.3
```

Comment:
We'll want websocket support in the Regal LS anyway at some point, so unless this needs to be a module of its own, let's just add this to the main module.

Reply:
(re-iterating the previous point because I've replied out-of-order) My assumption was that to use Regal via WS, you'd integrate the websocket handling in your (Go) application in some way. The new package allows you to start an LSP server per websocket session, which happens (😎) to be what the next playground iteration uses. It's also a bit of multi-tenant that way, since the file system doesn't really have to come into play.

How best to expose a WS from the regal binary is another question. Happy to help but I don't know the best design for this yet. Share an LSP session the current working directory? Start a fresh one...? 🤔